Readme.md

```diff
@@ -1,31 +1,31 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128579525/22.1.3%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E331)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* [Form1.cs](./CS/CustomSortingExample/Form1.cs) (VB: [Form1.vb](./VB/CustomSortingExample/Form1.vb))
-<!-- default file list end -->
-# Pivot Grid for WinForms - How to use a hidden field to sort the visible field
+# Pivot Grid for WinForms - How to Use a Hidden Field to Sort the Visible Field Values
 
-This example demonstrates how to sort a field (_Sales Person)_ by the hidden data field values (_Last Name_).
+This example demonstrates how to sort the _Sales Person_ field by the hidden _Last Name_ data field.
 
 Check the _Enable custom sorting_ box to sort the _Sales Person_ values by _Last Name_ instead of the default alphabetical sorting order.
```

Comment:
```suggestion
Check the _Enable custom sorting_ box to sort the _Sales Person_ values by _Last Name_ instead of the default alphabetical sort order.
```